cmd/kubeless/route/route.go

```diff
@@ -36,3 +42,60 @@ func init() {
 
 	}
 }
+
+func getRouteDefinition(funcObj spec.Function, funcName, hostName, ns string, enableTLSAcme bool) (v1beta1.Ingress, error) {
+	port := intstr.FromInt(8080)
+	if len(funcObj.Spec.ServiceSpec.Ports) != 0 {
+		port = funcObj.Spec.ServiceSpec.Ports[0].TargetPort
+	}
+
+	if port.IntVal <= 0 || port.IntVal > 65535 {
+		return v1beta1.Ingress{}, fmt.Errorf("Invalid port number %d specified", port.IntVal)
+	}
+
+	ingress := v1beta1.Ingress{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      funcName,
+			Namespace: ns,
+			Labels:    funcObj.Metadata.Labels,
+		},
+		Spec: v1beta1.IngressSpec{
+			Rules: []v1beta1.IngressRule{
+				{
+					Host: hostName,
+					IngressRuleValue: v1beta1.IngressRuleValue{
+						HTTP: &v1beta1.HTTPIngressRuleValue{
+							Paths: []v1beta1.HTTPIngressPath{
+								{
+									Path: "/",
```

Comment:
I would leave this `Path` as a parameter, that way we can easily extend it to support different paths (even if `"/"`is hardcoded in the caller for the moment). That will be needed for some examples in the `Serverless` plugin.